pradeep-rajapaksha/laravel-coding-test-level-1
# Third Test - Advance Topic

### What does this pull request fix/change?
This PR is for implementing Redis caching, sending emails on event create, user authentication 

### Testing steps
- Pull down this branch
- Run `composer install` and `php artisan server`
- Go to http://127.0.0.1:8000/events 
- Check that each and every functionality working correctly.
